README.rst

```diff
@@ -52,6 +60,19 @@ Examples_ that demonstrate how to use it.
 .. _Examples: https://skypy.readthedocs.io/en/stable/examples/index.html
 
 
+Get in Touch
+------------
+
+You are welcome to get in touch about the SkyPy package and code
+using our `Discussions page`_.
+
+Or for any other questions about the project in general, please get in touch with
+the current SkyPy co-ordinators:
```

Comment:
```suggestion
the SkyPy co-ordinators (current co-ordinators Richard Rollins and Nicolas Tessore):
```

Reply:
Do we really need to put names down? Coordinators will be rotating from time to time and that would mean updating their names here.

Reply:
Also Adam and Lucia. And for the sake of not having to update this list again in two weeks I'm happy to be left off.

Reply:
I agree with @Lucia-Fonseca and would prefer to leave names off, so that things don't get out of sync or confused ((as it seems they have already done?)

Reply:
My idea was to give people an idea who they are talking to. But I can see that it will likely get out of sync.